docs/en/observability/observability-introduction.asciidoc

```diff
@@ -2,77 +2,115 @@
 [role="xpack"]
 = What is Elastic {observability}?
 
-{observability} provides you with granular insights and context into the behavior
-of applications running in your environments. At Elastic, we view observability as an
-attribute of any system that you build and want to monitor. Being able to detect
-and fix root cause events quickly within an observable system is what we consider
-a minimum requirement for any analyst.
-
-https://www.elastic.co/observability[Elastic {observability}] provides you with a
-single stack to unify your logs, infrastructure metrics, uptime data, application traces, user experience data, and synthetics.
-Ingest your data directly to {es}, where you can further process and enhance the data,
-before visualizing it in {kib}.
-
-image::images/observability.png[Elastic {observability}]
+{observability} provides granular insights and context into the behavior of applications running in your environments. 
+It's is an important part of any system that you build and want to monitor. 
+Being able to detect and fix root cause events quickly within an observable system should be a minimum requirement for any analyst.
 
 Search, monitor, and apply analytics in real time to events happening across all of
```

Comment:
I know you didn't write this, but the shift into imperative mood is a little abrupt here. I also wonder if this paragraph might make more sense _after_ you introduce Elastic Observability.  You might even want to break it up into a bulleted list. You could even use the list to introduce the sections that follow.

So I would try moving para 2 after para 3. And maybe begin the new para 3 with _something_ like:

```
For example, in Kibana you can search, monitor, and apply....
```

Another option, which might even be better, is to just remove the 2nd para completely because I suspect the sections that follow cover all these details pretty well.